packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,295 @@
+| validationState                                                                                   | `error` , `success`                                                                                                                                                                                                                                    | No       |                                                           | Applies the style to the input field based on validation status and shows either `errorText` or `successText` respectively                                                                                                                                                            |
+| helpText                                                                                          | `string`                                                                                                                                                                                                                                               | No       |                                                           | Shown when we want to add some hint to the input field. Displayed under the input field. Only one of `helpText`, `errorText` or `successText` is shown at a time in the priority order as `errorText`, `successText`, `helpText`                                                      |
+| errorText                                                                                         | `string`                                                                                                                                                                                                                                               | No       |                                                           | Shown when the `validationState` of the input field is set to Error. Only one of `helpText`, `errorText` or `successText` is shown at a time in the priority order as `errorText`, `successText`, `helpText`                                                                          |
+| successText                                                                                       | `string`                                                                                                                                                                                                                                               | No       |                                                           | Shown when the `validationState` of the input field is set to Success. Only one of `helpText`, `errorText` or `successText` is shown at a time in the priority order as `errorText`, `successText`, `helpText`                                                                        |
```

Comment:
**Question:** Why no option of warningState?

Reply:
Well, I don't know whether it's implemented or not, but in certain places in our system amount is accepted in paise and in some cases it's in Rupees, so can be used in one of those cases to warn the users

Reply:
we'll enhance later on then once we have a use case